pages/asanas/[id].js

```diff
@@ -44,7 +44,7 @@ export default function Asana({ asana }) {
 }
 
 const StyledArticle = styled.article`
-  border-radius: 32px;
+  border-radius: 2rem;
```

Comment:
Are you sure you wanna make the radius depending on the root font size? The larger it gets, the rounder your boxes will be. :) I would suggest to keep pixels.

Reply:
Yes, true. Doesn't make much sense. I changed it back 😅